discovery-provider/plugins/notifications/src/__tests__/mappings/create.test.ts

```diff
@@ -36,16 +35,6 @@ describe('Create Notification', () => {
 
   afterEach(async () => {
     await resetTests(processor)
-    jest.clearAllMocks()
-    await processor?.close()
-    const testName = expect
-      .getState()
-      .currentTestName.replace(/\s/g, '_')
-      .toLocaleLowerCase()
-    await Promise.all([
-      dropTestDB(process.env.DN_DB_URL, testName),
-      dropTestDB(process.env.IDENTITY_DB_URL, testName)
-    ])
```

Comment:
i see this removed in some places but not added back anywhere, should this be a part of `resetTests`? or are we fine with dangling test dbs not being cleared out?

Reply:
this logic is all in resetTests so this was duplicate / unnecessary

Reply:
https://github.com/AudiusProject/audius-protocol/blob/9712f84a238addfaa36585c941905835fdb5bd62/discovery-provider/plugins/notifications/src/utils/populateDB.ts#L114

Reply:
sweet!